frontend/src/views/Layout/DashboardLayout_gdd.spec.js

```diff
@@ -83,7 +83,7 @@ describe('DashboardLayoutGdd', () => {
       })
 
       it('has first item "send" in navbar', () => {
-        expect(navbar.findAll('ul > li').at(0).text()).toEqual('sent')
+        expect(navbar.findAll('ul > li').at(0).text()).toEqual('send')
```

Comment:
How did this get into master?

Reply:
The Test is not yet required, fix it and we can make that happen